src/components/molecules/MessageList.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="bg-gray-infolt p-6 space-y-6 overflow-auto"> 
+        <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+         <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+         <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+        <message-card />
+        <message-card title="Empty box above" />
```

Comment:
line 19 and 18, Not sure if we need those? 

Also, I think it would be good to follow the Figma and only have 1 message card. (You can leave a comment in the page stating that multiple cards can be added to the list. Perhaps leave a message card commented.)